configs/constants.go

```diff
@@ -4,4 +4,5 @@ const (
 	WORKFLOW_FILE = "configs/workflows.json"
 	SCHEDULE_FILE = "configs/schedule.json"
 	LOGGING_FILE  = "log"
+	MAX_AGE_TO_PROCESS_WIN_EVTS_IN_DAYS = 1
```

Comment:
This feature is also not necessary; since we're skipping on the Bookmark Feature we will only be handling events as they happen, never looking back. Therefore, it's impossible for a win event's age to be more than a few seconds

Reply:
I see, done.